thirdeye/thirdeye-frontend/app/pods/services/api/anomalies/service.js

```diff
@@ -25,6 +25,9 @@ const HumanizedAnomaly = EmberObject.extend({//TODO: rename `anomaly` to `record
   duration: computed('anomaly.{start,end}', function() {
     return getFormatedDuration(get(this, 'anomaly.start'), get(this, 'anomaly.end'));
   }),
+  alertQueryUrl: computed('anomaly.{queryDuration,queryStart,queryEnd}', function() {
```

Comment:
If the #link-to works, above, this would probably not be needed.

Reply:
True. I tried a few times to use the #link-to, however can't seem to figure out how to make it work with this pattern. If you can figure it out, let me know. Thanks!

Reply:
Worst case i can make a custom link-to wrapper helper.